Shared/API/Onboarding+Cloud.swift

```diff
@@ -82,11 +83,18 @@ struct FinalizeOnboarding: CloudFunction {
         let params: [String: Any] = ["passId": self.passId,
                                      "reservationId": self.reservationId]
         
-        return try await self.makeRequest(andUpdate: statusables,
-                                          params: params,
-                                          callName: "finalizeUserOnboarding",
-                                          delayInterval: 0.0,
-                                          viewsToIgnore: viewsToIgnore)
+        _ = try await self.makeRequest(andUpdate: statusables,
+                                       params: params,
+                                       callName: "finalizeUserOnboarding",
+                                       delayInterval: 0.0,
+                                       viewsToIgnore: viewsToIgnore)
+
+        guard let user = User.current() else {
+            throw ClientError.message(detail: "No user found.")
+        }
+
+        // Refresh the user so it's activation status is properly reflected.
+        return try await user.fetchInBackground()
```

Comment:
The user was still "inactive" locally so the main coordinator prevented them from proceeding.

Reply:
ah nice catch.